frontend/src/app/libs/common-components/components/input/input-material.scss

```diff
@@ -0,0 +1,29 @@
+@import "variables";
+
+.input-component .mat-form-field-flex{
```

Comment:
missing empty space before `{` check everywhere

Reply:
also rename this file to input.theme.scss and import it in `app-theme.scss` sample here https://github.com/GandalfClub/gandalf-app/pull/67/files